libmamba/src/api/install.cpp

```diff
@@ -106,10 +115,11 @@ namespace mamba
             specs_f << d.c_str() << '\n';
         specs_f.close();
 
-        replace_all(install_instructions, "{0}", specs.path());
+        for (auto& arg : install_instructions)
```

Comment:
I'm not totally sure this is the right way to do this replacement. Would it be ok to just replace the whole string it it is equal to `"{0}"`, or other simpler mechanism?
Maybe it would simpler to pass the value into `get_other_pkg_mgr_install_instructions` so that the instructions already contain the path.